version.json

```diff
@@ -0,0 +1,5 @@
+{
+  "publisher": {
+    "version": "v1.1.35-SNAPSHOT"
```

Comment:
I think this is the old release version format. From https://github.com/HL7/fhir-ig-publisher/releases/, it looks like the release versions are just semantic version numbers.

Reply:
Changed to `"1.1.35"`!